controllers/generateConfigController.js

```diff
@@ -0,0 +1,190 @@
+const fs = require("fs");
+const { validEngines } = require("../utils/translationEnginesUtils");
+const { confirmUserAction, promptUserInput } = require("../utils/promptUtils");
+const { parsePath } = require("../utils/getConfigPath");
+const { listFilesOnDirectory } = require("../utils/listFiles");
+const { supportedLanguagesCodes } = require("../utils/supportedLanguagesUtils");
+
+const _promptTranslationEngines = () => {
+  const translationEngines = [...validEngines];
```

Comment:
I notice you are not changing this array, why create a copy? Maybe it is enough to use validEngines directly

Reply:
You're right, will use it directly